lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -114,7 +117,7 @@ class DiscussionEntity(Enum):
     comment = 'comment'
 
 
-def _get_course(course_key, user):
+def _get_course(course_key, user, check_tab=True):
```

Comment:
Please explain about the kwarg `check_tab` in the docstring.

Reply:
Done.